pkg/computenode/shard_fsm.go

```diff
@@ -358,7 +358,11 @@ func runningState(ctx context.Context, m *shardStateMachine) StateFn {
 	// we get a "proposal" from this method which is not the results
 	// but what the compute node verifier wants to pass to the requester
 	// node verifier
-	proposal, err := m.node.RunShard(ctx, m.Shard)
+
+	// Need to do something with RunOutput
```

Comment:
Is this still true? Does the stdout and stderr make it to describe output?

Reply:
no not yet sadly - need to work with @wdbaruni to understand how to get into state machine